Psychtoolbox/PsychOneliners/BackupCluts.m

```diff
@@ -31,6 +32,7 @@ function BackupCluts(screenIds)
 if isempty(screenIds)
     screenIds = Screen('Screens');
 end
+screenIds(screenIds==0) = []; % remove screens with 0 index
 
```

Comment:
That won't work on a Windows single display setup, where screenIds == 0 for the one single monitor.
You'd need something like the following to only handle Windows + multi-screen + screen 0

if IsWin && length(Screen('Screens')) > 1
    screenIds(screenIds==0) = []; % remove screens with 0 index
end

Reply:
OK, fixed.